.github/workflows/corpus.yml

```diff
@@ -7,7 +7,11 @@ on:
 
 jobs:
   build:
+    defaults:
+      run:
+        shell: bash
```

Comment:
Setting `bash` here invokes the default `-o pipefail` behavior that correctly makes it so that the `go test -count=1 -v -tags corpus ./integration_tests/... | tail` fails when the tests fail.